src/components/application_manager/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -226,20 +226,18 @@ void PerformInteractionRequest::on_event(const event_engine::Event& event) {
     }
     case hmi_apis::FunctionID::UI_PerformInteraction: {
       LOG4CXX_DEBUG(logger_, "Received UI_PerformInteraction event");
-      ui_response_recived_ = true;
+      ui_response_received_ = true;
       unsubscribe_from_event(hmi_apis::FunctionID::UI_PerformInteraction);
       ui_result_code_ = static_cast<hmi_apis::Common_Result::eType>(
           message[strings::params][hmi_response::code].asUInt());
       GetInfo(
           HmiInterfaces::HMI_INTERFACE_UI, ui_result_code_, message, ui_info_);
-      if (ProcessUIResponse(event.smart_object(), msg_param)) {
-        return;
-      }
+      ProcessUIResponse(event.smart_object(), msg_param);
```

Comment:
@AGritsevich , why you call event.smart_object() instead of using 'message' variable?